src/posts/feasibility.md

```diff
@@ -0,0 +1,122 @@
+---
+title: The feasibility of a Pause
+description: Is a pause actually possible?
+---
+<!-- does feasibility alludes to some amount of resources being enough for the objective or not? I was going to go with "possibility" for a while but it sounded worse -->
+<!-- I almost didn't put sources at all, ideally that should change -->
+
+One of the most common counterarguments for a pause is not that a it would be bad, but that there's no way we'll achieve it. 
```

Comment:
I think it may be more powerful to start off with some "AGI is inevitable" type of remarks. 

See this tweet for example: https://twitter.com/PauseAI/status/1763311084771938483